scripts/cpplint.py

```diff
@@ -6516,6 +6516,9 @@ def ProcessFile(filename, vlevel, extra_check_functions=[]):
   if Search(r'_builtin_headers(_[a-z0-9_-]+)?\.h$', filename):
     return
 
+  if Search(r'^regression/.*\.cpp', filename):
+    return
+
```

Comment:
Is there extra work required to make the CI job pick up this change, or did we somehow get it wrong?